common/models.py

```diff
@@ -72,6 +85,7 @@ def path_to_code(path):
         return os.path.relpath(path, os.path.abspath("./tasks"))
 
     def dir(self):
+        print(os.getcwd())
```

Comment:
This is probably not needed.

Reply:
Ah, this is a local change that I used to debug the weird getcwd changes, thanks.